documentation/operators/src/nodes/gateway-setup.md

```diff
@@ -2,6 +2,10 @@
 
 > The Nym gateway was built in the [building nym](../binaries/building-nym.md) section. If you haven't yet built Nym and want to run the code, go there first.
 
+
+```admonish info
+As a result of [Project Smoosh](../faq/smoosh-faq.md), the current version of `nym-gateway` binary also contains `nym-network-requester` function which can be turned on or off [by the operator](./gateway-setup.md#initialising-gateway-with-network-requester). This combination is a basis of Nym exit gateway node - an essential piece in our new setup. Please read more in our [Project Smoosh FAQ](../faq/smoosh-faq.md) and [Exit Gateways Page](../legal/exit-gateway.md). We recommend operators to start shifting to that model instead of running gateway and network requester as two separated binaries.
```

Comment:
* "function" -> "functionality" imo 
* "on or off" -> "enabled" 
* "We recommend operators to start shifting to that model instead of running gateway and network requester as two separated binaries." -> "We recommends operators begin to shift their setups to this new combined node, instead of operating two separate binaries"

Reply:
all done